sdc/crypto/exceptions.py

```diff
@@ -0,0 +1,23 @@
+
+class CryptoError(Exception):
+    ''' General Exception raise by this library if an error occurs not covered below '''
+    pass
+
+
+class MissingKidException(Exception):
+    ''' Raised if a Kid cannot be found in the keystore '''
```

Comment:
Isn't this raised if the Kid is not found in the message header rather than in the keystore?